apps/assessment_registry/management/commands/migrate_old_assessments.py

```diff
@@ -332,6 +332,15 @@ def _get_bg_crisis_type():
 
         }
         assessment_reg, created = AssessmentRegistry.objects.get_or_create(**input_data)
+        assessment_reg.save()
+
+        # Update the new assessment's create date, update date to old date
+        AssessmentRegistry.objects.update(
+            created_at=models.F('lead__assessment__created_at'),
+            created_by=models.F('lead__assessment__created_by'),
+            updated_at=models.F('lead__assessment__updated_at'),
+            updated_by=models.F('lead__assessment__updated_by'),
+        )
```

Comment:
Let's use this like `migrate_cna_questions()`. Running this for every Assessment Registry copy doesn't make sense.